spec/factories/users.rb

```diff
@@ -4,6 +4,6 @@
     name     { Faker::Name.name }
     password '12345678'
     password_confirmation '12345678'
-    admin true
+    role 1
```

Comment:
I don't understand why we are using integer literals to refer to the role names.

My understanding of the Rails `enum` type is that it hides the underlying numeric field type used in the database and  instead lets us use meaningful/readable named values within the code. E.g. http://www.justinweiss.com/articles/creating-easy-readable-attributes-with-activerecord-enums/

Can this and the other literals not be changed to `role 'admin' or `subject.role = 'admin'`? That is far easier to understand from a maintainer/reviewer perspective.

If we use integer literals in our code then we've completely lost all the benefits we could have had.

Maybe I've misunderstood something, but sorry this seems confusing.